src/pages/CharacterDetailPage.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import CharacterDetail from '../components/app/characters/CharacterDetail';
+
+const CharacterDetailPage = () => (
+  <CharacterDetail />
+);
```

Comment:
You either need to use your hook here and pass the character down as props using a spread operator or use the hook in your characterdetail component to get that info. That's why the deets aren't rendering on your detail page